src/ServiceManagement/Sql/Commands.SqlDatabase/Database/Cmdlet/StartAzureSqlDatabaseImport.cs

```diff
@@ -66,7 +64,7 @@ public class StartAzureSqlDatabaseImport : SqlDatabaseCmdletBase
             ParameterSetName = ByContainerObjectParameterSet,
             HelpMessage = "The Azure Storage Container to place the blob in.")]
         [ValidateNotNull]
-        public AzureStorageContainer StorageContainer { get; set; }
+        public CloudBlobContainer StorageContainer { get; set; }
```

Comment:
This will be a breaking change for this cmdlet.  What are the differences in creating a CloudBlobContainer versus an AzureStorageContainer?